tests/system/test_system.py

```diff
@@ -443,6 +446,58 @@ def test_create_database_pitr_success(self):
             for result in results:
                 self.assertEqual(result[0], retention_period)
 
+    @unittest.skipIf(
+        USE_EMULATOR, "Default leader setting is not supported by the emulator"
+    )
+    def test_create_database_with_default_leader_success(self):
+        pool = BurstyPool(labels={"testcase": "create_database_default_leader"})
+
+        # Create a multi-region instance
+        multi_region_config = "nam3"
+        ALT_INSTANCE_ID = "new" + unique_resource_id("-")
+        config_name = "{}/instanceConfigs/{}".format(
+            Config.CLIENT.project_name, multi_region_config
+        )
+        instance = Config.CLIENT.instance(
+            instance_id=ALT_INSTANCE_ID, configuration_name=config_name
```

Comment:
Please add the labels used by the other instance, see [here](https://github.com/googleapis/python-spanner/blob/e939fa7fbf5472f117f7150c5bd54fb0ccdc9810/tests/system/test_system.py#L151-L156). They are used to clean up the instance if there is a resource leak, e.g. if the testing is interrupted before clean up can happen.